src/composables/use-backdrop.ts

```diff
@@ -36,6 +36,6 @@ export const useBackdrop = (show: Ref<boolean>, { scrollable }: IBackdropOptions
         }
       }
     },
-    { lazy: true },
+    // { lazy: true },
```

Comment:
I commented this line out because it was giving me an error in my IDE (VSCode). It might be fine to add back in.